src/pages/Books/Books.js

```diff
@@ -35,24 +35,24 @@ const Books = () => {
     const [totalPages, setTotalPages] = useState(0);
 
     useEffect(()=>{
-        getBooks(page, genres, sort).then(res=>{
+        api.getBooks(page, genres, sort).then(res=>{
             setBooks(res.books);
             setTotalPages(Math.ceil(res.total/BOOKS_PER_PAGE));
         });
     },[page]);
 
     const pageHandler = (event, value) => {
         setPage(value);
-        getBooks(page, genres, sort).then(res=>{
+        api.getBooks(page, genres, sort).then(res=>{
             setBooks(res.books);
         });
     }
 
-    const sortHandler = (event) => {
+    const sortHandler = event => {
         setSort(event.target.value);
     }
 
-    const genresHandler = (event) => {
+    const genresHandler = event => {
         const {
             target: {value},
         } = event;
```

Comment:
both of these are using `event.target.value` but are inconsistent in implementation. So either put something like this `genresHandler = ({ target: { value } })` for both, or in the place you are using the handler, put `onChange={e => genresHandler(e.target.value)}`